o2.sh

```diff
@@ -134,44 +134,10 @@ fi
 unset DYLD_LIBRARY_PATH
 cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                                        \
       ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"}                                             \
-      -DCMAKE_MODULE_PATH="$SOURCEDIR/cmake/modules;$FAIRROOT_ROOT/share/fairbase/cmake/modules;$FAIRROOT_ROOT/share/fairbase/cmake/modules_old"  \
-      -DFairRoot_DIR=$FAIRROOT_ROOT                                                         \
-      -DALICEO2_MODULAR_BUILD=ON                                                            \
-      -DROOTSYS=$ROOTSYS                                                                    \
-      ${PYTHIA6_ROOT:+-DPythia6_LIBRARY_DIR=$PYTHIA6_ROOT/lib}                              \
-      ${GEANT3_ROOT:+-DGeant3_DIR=$GEANT3_ROOT}                                             \
-      ${GEANT4_ROOT:+-DGeant4_DIR=$GEANT4_ROOT}                                             \
-      ${VGM_ROOT:+-DVGM_DIR=$VGM_ROOT}                                                      \
-      ${GEANT4_VMC_ROOT:+-DGEANT4_VMC_DIR=$GEANT4_VMC_ROOT}                                 \
-      -DFAIRROOTPATH=$FAIRROOT_ROOT                                                         \
-      ${BOOST_ROOT:+-DBOOST_ROOT=$BOOST_ROOT}                                               \
-      ${DDS_ROOT:+-DDDS_PATH=$DDS_ROOT}                                                     \
-      -DZMQ_DIR=$ZEROMQ_ROOT                                                                \
-      -DZMQ_INCLUDE_DIR=$ZEROMQ_ROOT/include                                                \
-      ${ALIROOT_VERSION:+-DALIROOT=$ALIROOT_ROOT}                                           \
-      ${PROTOBUF_ROOT:+-DProtobuf_LIBRARY=$PROTOBUF_ROOT/lib/libprotobuf.$SONAME}           \
-      ${PROTOBUF_ROOT:+-DProtobuf_LITE_LIBRARY=$PROTOBUF_ROOT/lib/libprotobuf-lite.$SONAME} \
-      ${PROTOBUF_ROOT:+-DProtobuf_PROTOC_LIBRARY=$PROTOBUF_ROOT/lib/libprotoc.$SONAME}      \
-      ${PROTOBUF_ROOT:+-DProtobuf_INCLUDE_DIR=$PROTOBUF_ROOT/include}                       \
-      ${PROTOBUF_ROOT:+-DProtobuf_PROTOC_EXECUTABLE=$PROTOBUF_ROOT/bin/protoc}              \
-      ${GSL_ROOT:+-DGSL_DIR=$GSL_ROOT}                                                      \
-      ${PYTHIA_ROOT:+-DPYTHIA8_INCLUDE_DIR=$PYTHIA_ROOT/include}                            \
-      ${HEPMC3_ROOT:+-DHEPMC3_DIR=$HEPMC3_ROOT}                                             \
       ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE}                             \
       ${ALIBUILD_O2_TESTS:+-DENABLE_CASSERT=ON}                                             \
-      -DMS_GSL_INCLUDE_DIR=$MS_GSL_ROOT/include                                             \
```

Comment:
we made this name up, IIRC, does it really find it?

Reply:
yes, using the right combination of find_package call `(find_package(ms_gsl MODULE))` and corresponding [find module](https://github.com/AliceO2Group/AliceO2/blob/dev/dependencies/Findms_gsl.cmake)

(but now that we're using MS_GSL_ROOT, we might as well consider ditching it...)